trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/index.ts

```diff
@@ -9,29 +9,61 @@ import {
   mapKeyingSheetToKeyingSheetViewModel,
 } from '../helpers';
 import { UtilisationReportReconciliationForReportViewModel } from '../../../types/view-models';
-import { validateFacilityIdQuery } from './validate-facility-id-query';
-import { getAndClearFieldsFromRedirectSessionData } from './get-and-clear-fields-from-redirect-session-data';
 import { FeeRecordPaymentGroup } from '../../../api-response-types';
+import { extractQueryAndSessionData } from './extract-query-and-session-data';
+
+export type GetUtilisationReportReconciliationRequest = CustomExpressRequest<{
+  query: {
+    facilityIdQuery?: string;
+    selectedFeeRecordIds?: string;
+  };
+}>;
+
+const clearRedirectSessionData = (req: Request): void => {
```

Comment:
I think these 3x delete lines should be moved into the main function, instead of passing `req` around

Reply:
Moved into the main function as suggested